modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -11,7 +11,14 @@ AWS Security Token Service (AWS STS) is a global web service that provides short
 
 [IMPORTANT]
 ====
-Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the Restic service is disabled. Use native snapshots to backup volumes. See _Known Issues_ for more information.
+Restic and Kopia are not supported in the OADP on ROSA with AWS STS environment. Make sure that the Restic/Kopia node agent is disabled. Use native snapshots to back up volumes. See _Known Issues_ for more information.
+====
+
+[IMPORTANT]
+====
+In an Amazon ROSA cluster using STS authentication, restoring backed-up data in a different AWS region is not supported.
```

Comment:
```suggestion
ROSA clusters with STS do not support moving backed-up data across AWS regions. 

```

Reply:
Hi @sbeskin-redhat,
   Hope all is well! Looking at the JIRA ticket that supports this PR,  I am not sure if what is being said is whether the data can be backed up across regions but then can't be restored, OR it can't be backed up at all. Seems that if it can't be restored from a different region then where the cluster is housed, then it shouldnt be able to be backed up to a different region in the first place? Sorry if I am making it more confusing then it needs to be!

Reply:
IIRC we're saying if your credential is STS authentication, the token used can only accessed (or at least in a way that we support) resources in that region. And so we can only upload backed up data to that region's object storage.

If you want to restore from another region, you have to on your own (which we are providing support) find a way to move data from objectStorageRegionA to objectStorageRegionB and use another regionB_STS_Credentials to restore.

Reply:
@mletalie 
"...do not support moving backed-up data across AWS regions." - this is not exactly correct. Data can be moved across regions using native AWS tools. But OADP does not support restoring backed-up data in an AWS region different from the one in which it was backed up.  Data can only be backed up and then restored within the same AWS region.